### DISCUSSION TITLE
Make Global Collections Manually Sortable

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#7813] [Feature Requests]
We can still allow for Role overrides, but this has the benefits of allowing global (across all roles) reordering of collections in both the nav sidebar **and the collections listing page**. I'd love a way to include grouping into the global data model too (within Settings)!

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
